File: trunk/test-scala/src/test/scala/com/hongkailiu/test/app/entity/PersonTest.scala
package com.hongkailiu.test.app.entity


import org.scalatest.{BeforeAndAfter, FunSuite}
import org.mockito.Mockito
/**
 * Created by hongkailiu on 2015-04-13.
 */
class PersonTest extends FunSuite with BeforeAndAfter {



  test("aaa"){
    val mockCalc:Calculator = Mockito.mock(classOf[Calculator])
    val vick = Person("Vick","Mackey",mockCalc)
    Mockito.when(mockCalc.add(1,2)).thenReturn(3);
    val m = vick.add(1,2)
    assert(3 === m)
  }

  before {
    println("before")
  }

  after {
    println("after")
  }
}

File: trunk/test-gradle-java/src/test/java/tk/hongkailiu/test/entitiy/PersonTest.java
package tk.hongkailiu.test.entitiy;

import org.junit.Assert;

import org.junit.Before;
import org.junit.Test;

/**
 * Created by ehongka on 4/6/16.
 */
public class PersonTest {

    private Person unitUnderTest;

    @Before public void setup(){
        unitUnderTest = new Person();
    }

    @Test public void testGetFirstname() throws Exception {
        unitUnderTest.print();
    }

    @Test
    public void testPrintMessage() {
        System.out.println("Inside testPrintMessage()");
        Assert.assertEquals(1, 1);
    }
}
